scripts/openzfs-merge.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+#
+# This script tries to merge OpenZFS commits to ZoL.
+#
+# Instruction:
+# 
+# Repository setup must be similar with openzfs-tracking.sh
+# requirements.
+#
+# Repository setup for valid compilation check:
+#	sh autogen.sh
+#	./configure --enable-debug
```

Comment:
Wouldn't it be safe to run `sh autogen.sh` and `./configure` for each commit? Is it because Makefile.in's don't often change? There are other .in files as well.